### DISCUSSION TITLE
React Native ou Xamarin?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1534] [Discussão]
Galera, sei que aqui é um grupo de React, provavelmente as respostas virão nesta linha...

Mas, aqui na empresa somos uma equipe focada em C# .Net Core no backend e React no front-end, chegamos a este dilema, qual a opinião de vocês?

Vejo muitos cases de react-native, vários projetos open, mas xamarin é bem menos... Não sei se levamos em conta a comunidade, ou pelo que eu vi o xamarim transpila para código nativo mesmo, isso talvez o faça mais performático, 
o que acham? O que escolheriam?
Conhecem algum benchmark ou material bom comparando ambos?

Muito obrigado, abraço!

**Comment by TecladistaProd** [comment]
Eu não êxito em escolher o React native. Questão de performance se comparado ao nativo e menor, mas dependendo do que for fazer o usuário nem percebe.
Libs, tutoriais, forums e a comunidade realmente é bem maior, sem contar que quanto ao xamarin eu desconheço mas o react native sempre tem atualizações e melhorias como o HermesJS para Android.

Sem contar que a própria Microsoft está investindo no React Native até mesmo para Windows, e tinha uma vez ouvido dizer que alguns apps mobile da mesma estava sendo escritos usando react native se eu não estiver enganado.

**Comment by wilsonneto-swe** [comment]
Opa @TecladistaProd , muito obrigado pelas informações, será que a MS ta usando React? hehe Vou procurar saber sobre, pessoalmente gosto bem mais do react-native também

**Comment by TecladistaProd** [comment]
@wilsonneto-dev , eles tem até uma documentação do sdk do react native no próprio site deles https://docs.microsoft.com/en-us/appcenter/sdk/getting-started/react-native
